ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/Invoker.java

```diff
@@ -53,16 +69,36 @@
  * @since 2.0.0
  */
 public abstract class Invoker extends DiagnosticReporter {
+    public static final String MODULE_NOT_FOUND_CODE = "BCE2003";
+    // TODO: After configurables can be supported, change this to that file location
```

Comment:
Shall we create an issue also and point the issue here along with the TODO? or can we fix this because the configurables are now supported

Reply:
The problem was what should be the config.toml to use. Because for shell, there is no specific config file.
Maybe we should have a different ShellConfig.toml file.? I added this todo to change the location if is happens.
I'll create an issue for this.